### PR TITLE
Add `REVOKE_REGISTRATION` and `REINSTATE_REVOKE_REGISTRATION` custom actions

### PR DESCRIPTION
**Pull request by cibelius**
## Description

Configure two new custom actions:
* REVOKE_REGISTRATION
* REINSTATE_REVOKE_REGISTRATION

Also fix icons on three workqueues.

## Checklist

- [x] I have linked the correct Github issue under "Development"
- [x] I have tested the changes locally, and written appropriate tests
- [x] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [x] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: src/form/v2/birth/index.ts [hunk 1]
```diff
@@ -243,6 +252,8 @@ export const birthEvent = defineConfig({
     ActionType.PRINT_CERTIFICATE,
     'ISSUE_CERTIFIED_COPY',
     ActionType.REQUEST_CORRECTION,
+    'REVOKE_REGISTRATION',
+    'REINSTATE_REVOKE_REGISTRATION',
```
[the remark this answers is not in the document]
Comment (cibelius) [reply src/form/v2/birth/index.ts]: The designs for these actions are coming from Jon and Ed: https://docs.google.com/spreadsheets/d/1V32XPsZfO6ONtCIBoUbhnL9JSqEoVnld0Meu8gnxGmw/edit?gid=337137562#gid=337137562
